WcaOnRails/app/models/post.rb

```diff
@@ -2,11 +2,37 @@
 
 class Post < ApplicationRecord
   belongs_to :author, class_name: "User"
+  has_many :post_tags, autosave: true, dependent: :destroy
 
   validates :title, presence: true, uniqueness: true
   validates :body, presence: true
   validates :slug, presence: true, uniqueness: true
 
+  def reload
+    @tags = nil
+    super
+  end
+
+  attr_writer :tags
+
+  def tags
+    @tags ||= post_tags.pluck(:tag).join(",")
```

Comment:
Is there something missing something with your suggestion? Where do we do the splitting on commas?